src/lsm/tree_fuzz.zig

```diff
@@ -339,19 +349,10 @@ pub fn run_fuzz_ops(fuzz_ops: []const FuzzOp) !void {
         allocator,
         Environment.size_max,
         Storage.Options{
-            // We don't apply storage faults yet, so this seed doesn't matter.
-            .seed = 0xdeadbeef,
             .read_latency_min = 0,
             .read_latency_mean = 0,
             .write_latency_min = 0,
             .write_latency_mean = 0,
```

Comment:
In forest_fuzz you set *_latency_mean. Do you want to do the same here? Or is there no way for reads/writes to interleave here?

Reply:
Good point; I'll update it. When there are multiple compactions (i.e. from different levels) running in parallel, then IOs will interleave.